static/js/util/util.js

```diff
@@ -272,14 +272,17 @@ export function getPreferredName(profile: Profile, last: boolean = true): string
  */
 export function getLocation(profile: Profile, showState: boolean = true): string {
   let { country, state_or_territory, city } = profile;
-  let subCountryLocation, countryLocation;
+  let subCountryLocation = city || "";
```

Comment:
why did you move this `subCountryLocation` statement out of the `else` block below?